plotsquared/configuration/worlds.md

```diff
@@ -114,17 +114,19 @@ worlds:
       min_height: -63
       # World border expands dynamically (false = disabled)
       border: false
+      # plot offset for the world-border around the claimed plots
```

Comment:
Added with [plotsquared#4213](https://github.com/IntellectualSites/PlotSquared/pull/4213). I hope this description is correct.